openpype/hosts/blender/scripts/build_workfile.py

```diff
@@ -205,33 +205,50 @@ def create_instance(creator_name, instance_name, **options):
     )
 
 
-def load_casting(project_name, shot_name) -> Set[OpenpypeContainer]:
-    """Load casting from shot_name using kitsu api."""
+def download_kitsu_casting(
+    project_name: str, shot_name: str, rig_only: bool = False
+) -> list:
```

Comment:
You can specify iterable items with hints like that
```suggestion
) -> List[dict]:
```